### ISSUE TITLE
Couldn't find a loader for given hwid and pkhash

### ISSUE THREAD

**Issue by hosamasd** [bkerler/edl#459]
HWID:              0x0011e0e100510000 (MSM_ID:0x0011e0e1,OEM_ID:0x0051,MODEL_ID:0x0000)
CPU detected:      "saipan"
PK_HASH:           0xa94e15342ec1d0d82ec6e65870301b1382e6559e8c5dd452cd4c86270e22439a1a766b8b593a8b92481e611c6ad73382
Serial:            0x14da5f8c

sahara - [LIB]: Couldn't find a loader for given hwid and pkhash (0011e0e100510000_a94e15342ec1d0d8_[FHPRG/ENPRG].bin) :(

**Comment by jakiki6** [comment]
it should be a "CPH2009EEA, Reno3 Pro, Find X2 Neo" according to the hash

https://github.com/hoplik/Firehose-Finder/blob/master/fh_collection/6AD73382/1/prog_firehose_ddr.elf should be the loader

**Comment by RenateUSB** [comment]
Yup, you're right. The hash and the hwid check out (in the elf itself).
I have to remember to check hoplik.
